sunpy/timeseries/sources/noaa.py

```diff
@@ -18,9 +18,169 @@
 from sunpy.util.metadata import MetaDict
 from sunpy.visualization import peek_show
 
-__all__ = ['NOAAIndicesTimeSeries', 'NOAAPredictIndicesTimeSeries']
+__all__ = ['NOAAIndicesTimeSeries', 'NOAAPredictIndicesTimeSeries', 'NOAAGoesSXRTimeSeries']
 
 
+class NOAAGoesSXRTimeSeries(GenericTimeSeries):
+    """
+    NOAA Goes soft X-ray (SXR) flux in near-real-time.
+    The NOAA Solar Weather Prediction Center (SWPC) provides near-real-time measurements
+    of the X-ray flux from GOES satelites.
+    The SXR flux as a function of time is used to track the solar activity and solar flares.
+    The near-real-time SXR data are provided from <https://services.swpc.noaa.gov/json/goes/>
+    in JSON format (updated every minute).
+    The solar SXR measurements are made in the 1-8 Angstrom (0.1-0.8 nm, short channel)
+    and 0.5-4.0 Angstrom (0.05-0.4 nm, short channel) passbands.
+    SWPC designates a Primary and a Secondary GOES Satellite for each instrument. Data from the
```

Comment:
Yes this is correct.
It might be better to replace it with the following:
"SWPC designates a Primary and a Secondary GOES Satellite (e.g. GOES-16/17) for each instrument, 
 the satellite from which the SXR measurement is made can be found in <https://services.swpc.noaa.gov/json/goes/instrument-sources.json>."

Reply:
This (https://services.swpc.noaa.gov/json/goes/instrument-sources.json) seems to indicate that 16 is the "primary" and 15 is the "secondary"?

Reply:
Yes according to the json this is correct, G16: primary G15: secondary for the sxr measurements.